todo-mvc/src/widgets/createTodoItem.ts

```diff
@@ -61,7 +65,14 @@ const createTodoItem = createWidgetBase
 							state: { classes: [ 'toggle' ], checked }
 						}),
 						d(createLabel, {
-							listeners: { dblclick: () => { todoEdit.do(state); } },
+							listeners: {
+								dblclick: () => { todoEdit.do(state); },
+								keypress: (event: KeyboardEvent) => {
+									if (event.which === 13) {
```

Comment:
to be fair they are both deprecated, so either is fine by me. `keyCode` will always have the value you want when it is a non printable key press (like `enter`) iirc